arcgis-android-toolkit/src/main/java/com/esri/arcgisruntime/toolkit/popup/util/Event.kt

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2021 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.esri.arcgisruntime.toolkit.popup.util
```

Comment:
I think this could move in a top level util package, for re-use by other components in the future.

```suggestion
package com.esri.arcgisruntime.toolkit.util
```

Reply:
updated